### ISSUE TITLE
When can we plot Treemap using bqplot?

### ISSUE THREAD

**Issue by xudong-z** [bqplot/bqplot#1217]
curious as titled. Believe I'm not the only one expecting to make interactive treemap within bqplot.

hope developers can advise. thanks!

**Comment by martinRenou** [comment]
cc. @ibdafna I remember you were talking about this

**Comment by ibdafna** [comment]
@martinRenou yes indeed! I actually started working on this a few months back but didn't finish. I'll add to the todo list.
